test/example_comment_created_body.json

```diff
@@ -0,0 +1,42 @@
+// Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License").
+// You may not use this file except in compliance with the License.
+// a copy of the License is located at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+{
```

Comment:
@collinpburns -- tried testing using this payload today. is this really what you get from JIRA? the `body` doesn't contain `issue_event_type_name`, which `jira_to_chime.py` expects on line 43. 

were you able to test it with this payload?